pipeline/config/exoscale/common-config.yaml

```
@@ -52,5 +52,7 @@ networkPolicies:
         - 0.0.0.0/0
   alertmanager:
     alertReceivers:
+  coredns:
```

I think you removed the 0.0.0.0 from the alertmanager thing above here
```suggestion
      ips:
        - 0.0.0.0/0
  coredns:
```

---

done